src/main/java/creativei/helper/ResponseHelper.java

```diff
@@ -127,35 +127,31 @@ public static InquiryVo getInquiryResponseById(Inquiry inquiry) {
     }
 
     public static List<InquiryVo> getAllInquiryResponse(List<Inquiry> inquiries) {
-        InquiryVo inquiryVo = new InquiryVo();
-        List<InquiryVo> inquiryVos = new ArrayList<>();
-        Iterator iterator = inquiries.iterator();
-        int i = 0;
-        while (iterator.hasNext()) {
-            inquiryVo.setAreaOfInterest(AreaOfInterest.enumToString(inquiries.get(i).getAreaOfInterest()));
-            inquiryVo.setName(inquiries.get(i).getName());
-            inquiryVo.setDob(LocalizationUtil.getFormattedDate(inquiries.get(i).getDob()));
-            inquiryVo.sethQualification(EducationQualification.enumToString(inquiries.get(i).getHighestEducation()));
-            inquiryVo.setEmail(inquiries.get(i).getEmail());
-            inquiryVo.setMobile(inquiries.get(i).getPhoneNumber());
-            inquiryVo.setId(inquiries.get(i).getId());
-            inquiryVo.setInquiryStatus(InquiryStatus.enumToString(inquiries.get(i).getInquiryStatus()));
-            inquiryVo.setInquiryDate(LocalizationUtil.getFormattedDate(inquiries.get(i).getInquiryDate()));
-            inquiryVo.setGender(Gender.enumToString(inquiries.get(i).getGender()));
-            inquiryVo.setComputerKnowledge(ComputerKnowledge.enumToString(inquiries.get(i).getComputerKnowledge()));
-            inquiryVo.setClosingStatus(FollowUpStatus.enumToString(inquiries.get(i).getClosingStatus()));
-            inquiryVo.setClosingSubStatus(FollowUpSubStatus.enumToString(inquiries.get(i).getClosingSubStatus()));
-            inquiryVo.setClosingRemark(inquiries.get(i).getRemark());
-            inquiryVo.setAddress(getCreateAddressResponseData(inquiries.get(i).getInquiryAddress(), new AddressVo()));
+        List<InquiryVo> inquiryVos=new ArrayList<>();
+        for(Inquiry inquiry:inquiries){
+            InquiryVo inquiryVo=new InquiryVo();
```

Comment:
1. Can we resuse the constructor InquiryVo(Inquiry)?
2. create reference outside the the loop, object creation can be done indide the loop. eg-
InquiryVo inquiryVo;
for(){
inquiryVo = new InquiryVo(Inquiry);
}

Reply:
@jitendra20  Is this fixed?

Reply:
No it is not fixed right now

Reply:
issue fixed in commit b6db493